File: 4_Matricies/transpose/transpose.js
// reverses each row of array
function transpose(array) {
    let temp;
    for (let y = 0; y < array.length; y++) {
      temp = array[y][0];
      array[y][0] = array[y][array.length - 1];
      array[y][array.length - 1] = temp;
    }
  }
  

File: 4_Matricies/transpose/transpose.c
// reverses each row of array
void transpose(int array[4][4], const int SIZE) {
    int temp;
    for (int x = 0; x < SIZE/2; x++)
        for (int y = 0; y < SIZE; y++) {
            temp = array[y][x];
            array[y][x] = array[y][SIZE-x-1];
            array[y][SIZE-x-1] = temp;   
        }
}